### DISCUSSION TITLE
STUdio:Impossible de glisser deposer les packs d'histoires

### DISCUSSION THREAD

**Discussion by cehanne** [marian-m12l/studio#319] [General]
Bonjour a tous
Je n'arrive pas a  avoir la petite main pour glisser deposer les packs sur la boite et inversement dans STUdio.
J'ai bien ma boite avec mes histoires qui apparait et j'ai bien des packs que j'avais réussi il ya longtemps a télécharger depuis ma boite.
Au lieu d'avoir la petite main j'ai un panneau d'interdiction comme sur la photo uniquement sur les pack notés "officiel".J'ai tout essayé  studio 0.3.1, 0.4.2, avec différents navigateurs, avec plusieurs version de Java JDK. Rien y fait.
Vous avez la soluce?
Merci d'avance !
Cedric
![Screenshot_20220920_132356](https://user-images.githubusercontent.com/23699930/191245945-85e3426f-94c5-4d01-a070-d9d3e60940e0.jpg)
![Screenshot_20220920_132329](https://user-images.githubusercontent.com/23699930/191245953-8f66c58f-1e6e-48d7-8adc-18e4cef8d072.jpg)

**Comment by ghost** [comment]
Bonjour,

Même soucis pour moi.

STUdio 0.3.1
Firmware 2.22


EDIT : voici la raison : https://github.com/marian-m12l/studio/issues/18
Position tout à fait honorable du dev ;-)

**Reply by Fikk78** [reply]
Bonjour,

J'ai exactement le même problème. @cehanne, @eds357, @dalotin, avez vous une solution.
J'ai lu le post Aptutech mais je ne vois toujours pas la solution.

Merci à l'avance pour votre aide.

Cordialement

**Comment by dalotin** [comment]
Bonjour, de ce fait, il est maintenant impossible de faire une sauvegarde de son appareil pour son usage personnel ?

**Comment by Fikk78** [comment]
Bonjour,

J'ai exactement le même problème. @cehanne, @eds357, @dalotin, avez vous une solution.
J'ai lu le post Aptutech mais je ne vois toujours pas la solution.

Merci à l'avance pour votre aide.

Cordialement
